github/apps_marketplace.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// MarketplaceService handles communication with the marketplace related
+// methods of the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/apps/marketplace/
+type MarketplaceService service
+
+// MarketplacePlan represents a GitHub Apps Marketplace Listing Plan.
+type MarketplacePlan struct {
+	URL                 *string   `json:"url"`
+	AccountsURL         *string   `json:"accounts_url"`
+	ID                  *int      `json:"id"`
+	Name                *string   `json:"name"`
+	Description         *string   `json:"description"`
+	MonthlyPriceInCents *int      `json:"monthly_price_in_cents"`
+	YearlyPriceInCents  *int      `json:"yearly_price_in_cents"`
+	PriceModel          *string   `json:"price_model"`
+	UnitName            *string   `json:"unit_name"`
+	Bullets             *[]string `json:"bullets"`
```

Comment:
For consistency with other structs that we map to GitHub responses, all of these should have `,omitempty` JSON tag option. E.g., see:

https://github.com/google/go-github/blob/511f540f1887d30b88cee4a2fcd1f2922754acf4/github/issues.go#L26-L52

It doesn't make a difference for unmarshaling JSON, but it does for marshaling. I don't think these will be marshaled (since we're receiving data from GitHub API, not sending these upstream). So, the only reason to do it is to maintain consistency.

See issue #537 for some more background information on this.

Reply:
I was wondering why they were there. I didn't realise that they had no impact on unmarshalling (but it makes perfect sense). Happy to maintain the consistency.